stardis/io/model/tests/test_model_io.py

```diff
@@ -8,57 +8,53 @@
 
 from stardis.io.model.marcs import read_marcs_model
 
-
 @pytest.fixture
-def marcs_model_test_data_file_path():
+def create_model():
     fname = "marcs_test.mod.gz"
     fpath = files("stardis.io.model.tests.data")
```

Comment:
Since you're editing this file, maybe handle file paths with the Pathlib module rather than importlib_resources. That's what we use in the rest of STARDIS. You can search for Pathlib in the github repo for examples.